lib/core/connection/pool.js

```diff
@@ -26,6 +26,8 @@ var CONNECTED = 'connected';
 var DESTROYING = 'destroying';
 var DESTROYED = 'destroyed';
 
+const CONNECTION_EVENTS = ['error', 'close', 'timeout', 'parseError', 'connect', 'message'];
```

Comment:
nit: `Set` (and swap `forEach`s for `for (const i of CONNECTION_EVENTS)`)

Reply:
done